ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS 2.x.x branch released xxxx-xx-xx
+
+Changes
+   * Fix possible memory leak in mbedtls_ssl_config_free().
```

Comment:
Memory leaks go in the "BugFixes" section, not "Changes". Given that the memory leak can only happen if users access `conf` structure fields directly, which is discouraged, I'm not sure how we should consider this one. I'd still say "BugFix", but then the ChangeLog entry should include some restriction like "that could occur only if users manipulated `conf->psk` and/or `conf->psk_identity` directly instead of using `mbedtls_ssl_conf_psk()` which is the only acceptable method for setting these fields".

Reply:
I disagree: since this only provides robustness against misuse, this should be in the “Changes” section.